backend/dynamic_parameter_service/dynamic_parameter_service.go

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
+
+// +build ui
+
+package dynamic_parameter_service
+
+import (
+	"fmt"
+    "io/ioutil"
+    "errors"
+    "os"
+
+    "github.com/gin-gonic/gin"
+    "github.com/omec-project/webconsole/backend/logger"
+)
+
+type route struct {
+    Pattern string
+	EnvVariable string
+}
+
+func AddDynamicParameterService(engine *gin.Engine) {
+	logger.WebUILog.Infoln("Adding dynamic parameters service")
+    group := engine.Group("/config/parameter")
```

Comment:
I think `inventory` would be a better endpoint name.

Along with my other comments here, we would make a POST call to `config/v1/inventory/gnb/<my gnb name>` to add a gnb,  a GET call to `config/v1/inventory/gnb` to get all available gnb's, a DELETE call to `config/v1/inventory/gnb/<my gnb name>` to delete a gnb.